index.bs

```diff
@@ -546,28 +568,93 @@ canvas.height = Math.max(leftEye.renderHeight, rightEye.renderHeight);
 </div>
 
 
-## VRStageParameters ## {#interface-vrstageparameters}
+## VRCoordinateSystem ## {#interface-vrcoordinatesystem}
+
+The VRCoordinateSystem interface defines a cartesian coordinate system measured in meters.  {{VRPose}} and {{VRFrameData}} matrices are always expressed in the context of a VRCoordinateSystem.
+The coordinate system uses these axis definitions:
 
-The {{VRStageParameters}} interface represents the values describing the the stage/play area for devices that support room-scale experiences.
+* Positive X is to the right of the origin.
+* Positive Y is up from the origin.
+* Positive Z is behind the origin.
 
 <pre class="idl">
-interface VRStageParameters {
-  readonly attribute Float32Array sittingToStandingTransform;
+interface VRCoordinateSystem {
+  boolean getTransformTo(VRCoordinateSystem coordinateSystem, Float32Array transform);
+};
+</pre>
+
+### Attributes ### {#vrcoordinatesystem-attributes}
+
+<dfn method for="VRCoordinateSystem">getTransformTo()</dfn>
+Fills in the transform to the supplied {{VRCoordinateSystem}} as a 16-element array containing the components of a 4x4 affine transformation matrix in column-major order.  Returns true if a transform exists between the origins of the two {{VRCoordianteSystem}} objects at the current frame's timestamp.  Returns false if the two {{VRCoordinateSystem}}s cannot be related for any reason (e.g. positional tracking loss).
+
+
+## VRAttachedFrameOfReference ## {#interface-vrattachedframeofreference}
+
+The VRAttachedFrameOfReference interface defines a reference frame that is body-locked to the {{VRDisplay}} from which it was created.  The {{VRAttachedFrameOfReference}}'s {{VRCoordinateSystem}} should be used for 3DOF experiences (such as 360 video) and as a fallback for other experiences when positional tracking is unavailable.
+
+This frame of reference has a fixed yaw relative to the user's surroundings that points in the direction the user was facing when the frame of reference was created.  The roll and pitch are relative to gravity.
+
+Data returned in calls to {{VRDisplay}}.{{getFrameData()}} using the reference frame's {{VRCoordinateSystem}} will be relative to that fixed orientation and may also include position data if the {{VRDisplay}} performs neck-modeling.
+
+<pre class="idl">
+interface VRAttachedFrameOfReference {
+  readonly attribute VRCoordinateSystem coordinateSystem;
```

Comment:
Yeah, I considered having a reset() function and ended up removing it because I didn't want to have two ways of accomplishing the same end result.
If we do just use create___FrameOfReference() to function as a "reset", do you have a scenario concern about the "origin" of the new FrameOfReference being the same as the previous one on Daydream devices?